docs/src/charm/howto/openstack.md

```diff
@@ -0,0 +1,120 @@
+# Integrating with OpenStack
+
+This guide explains how to integrate {{product}} with the OpenStack cloud
+platform.
+
+## Prerequisites
+
+To follow this guide, you will need:
+
+- An [OpenStack][openstack] cloud environment.
+- Juju set-up as the deployment tool for Openstack. Please refer to
+  [Juju and Openstack][juju-openstack] documentation for more information.
```

Comment:
Ehh --- this isn't REALLY necessary.  You just need openstack credentials  -- openstack doesn't have to be deployed BY juju